app/models/flexible_page/flexible_section/base.rb

```diff
@@ -0,0 +1,13 @@
+class FlexiblePage
+  module FlexibleSection
```

Comment:
I've never seen a module nested inside a class before. It looks a bit odd.

Reply:
I thought we were confined by rubocop to explicit nesting (in which case this is the only way we can do this), but I checked in landing pages and we were doing `module LandingPage::Block`, so I've used that style now.